src/main/java/com/battleship/backend/models/Game.java

```diff
@@ -57,9 +58,39 @@ public void removePlayerShip(Ship shipToRemove){
         playerShipList.remove(deletedShip);
     }
 
-    public void takeComputerTurn(){
+    public String takeComputerTurn(){
         Request hitRequest = computerPlayer.generateValidHitRequest();
         fire(getPlayerBoard(), hitRequest);
+        return shipHitResult(getPlayerBoard(), hitRequest);
+    }
+
+    public boolean isGameOver(){
+        return getPlayerBoard().isEveryShipSunk() || getComputerBoard().isEveryShipSunk();
+    }
+
+    public String getWinner(){
+        if(getPlayerBoard().isEveryShipSunk()){
+            return "Computer Player is the Winner!";
+        }else if(getComputerBoard().isEveryShipSunk()){
+            return "Human Player is the Winner!";
+        }else return "Game is not over";
+    }
+
+    //returns result of a hit request
```

Comment:
Is this comment necessary? The method name feels expressive enough.

Reply:
Good comments on comments.  I'm personally not a fan of comments as I feel they get stale quickly, and need a lot of maintenance (need to remember to update).

Reply:
Removed.